### PR TITLE
WIP: Add documentation for using vectors of trust

### PR DESCRIPTION
**Pull request by jmhooper**
This commit adds documentation to describe vectors of trust.

This commit does not remove ACR value documentation. ACR values will continue to be supported for some period of time while vectors of trust are rolled out.

This commit primarily describes how the feature works in OIDC.

### CODE REVIEW

File: _pages/oidc/authorization.md [hunk 1]
```diff
@@ -130,6 +130,10 @@ In an **unsuccessful authorization**, the URI will contain the parameters `error
       <p>
         Multiple values can be joined with a space (before being URI-escaped in the final URL).
       </p>
+      <p>
```
Comment (zachmargolis) [inline _pages/oidc/authorization.md:133]: maybe in another pass we can go add `markdown="1"` to the `<div>` surrounding this so we can author text changes like this in markdown

File: _pages/vot-trust-framework.md [hunk 2]
```diff
@@ -0,0 +1,62 @@
+---
```
Comment (jmhooper) [inline _pages/vot-trust-framework.md:1]: This filename here is intended to match up to the trustmark we have published in the repo: https://github.com/18F/identity-idp/blob/5837343cd6c1a9fe3f78d56d9931808e108e80a0/config/application.yml.default#L344

File: _pages/oidc/authorization.md [hunk 3]
```diff
@@ -139,10 +143,36 @@ In an **unsuccessful authorization**, the URI will contain the parameters `error
...
+      </div>
+    </div>
+    <div class="grid-row dev-doc-row">
+      <div class="grid-col-5">
+        <h4 id="vtr" class="parameters">vtm</h4>
+      </div>
+      <div class="grid-col-7">
+          A URL linking to the Trustmark for the <code class="language-plaintext highlighter-rouge">vtr</code> parameter.
+          This should be set to <a class="usa-link" href="{{ '/vot-trust-framework/' | prepend: site.baseurl }}">
+          https://developer.login.gov/vot-trust-framework</a>.
```
Comment (jmhooper) [inline _pages/oidc/authorization.md:170]: This URL is intentionally not configured to use the baseurl here. It needs to match the expected VTM value that the IdP knows about: https://github.com/18F/identity-idp/blob/5837343cd6c1a9fe3f78d56d9931808e108e80a0/config/application.yml.default#L344

File: _pages/vot-trust-framework.md [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+    href: "#using-vectors-of-trust"
+  - text: Authentication components
+    href: "#authentication-components"
+---
+
+## Using vectors of trust
+
+Vectors of trust are a combination of individual components. Each component describes a feature or requirement for the user during the transaction. For example, a component may specify that identity-proofing or that a particular MFA method must be used.
+
+Vector components are made up of a upper-case letter followed by a lower-case letter or a number. For example, `P1` or `C2`. A vector of trust is made by combining these components and arranging them into a period-separated list. For example, `C1.C2.P1.Pb`.
```
Comment (tahineemay) [inline _pages/vot-trust-framework.md:16]: Suggestion: the example "P1 or C2" shows the same combination, perhaps we should make it a letter and number and a letter and lower case letter.

Comment (jmhooper) [reply _pages/vot-trust-framework.md]: I just changed it to '"P1" or "Ca"'

File: _pages/oidc/authorization.md (hunk 3 above)
Comment (zachmargolis) [reply _pages/oidc/authorization.md:170]: Our canonical domain name for these docs is the plural one (developers), should we use that here (and in the VM claim?)

```suggestion
          https://developers.login.gov/vot-trust-framework</a>.
```

File: _data/errors.yml [hunk 5]
```diff
@@ -43,6 +43,16 @@ oidc:
...
         - Ensure that you are passing in the appropriate IAL Service Level value as well as any Authentication Assurance (AAL) values you might want for your application.
         More information about the `acr_values` parameter can be found here: [https://developers.login.gov/oidc/authorization/#acr_values](/oidc/authorization/#acr_values){:target="_blank"}
+    - title: No acceptable vots found
+      id: oidc-no-vot
+      accordion-id: no_vot
+      content: |
+        ##### Why it's happening
+        The vectors of trust found in the VTR parameter are not valid
+        ##### What to do:
+        - Check the `vot` parameter in your authentication request.
```
Comment (monfresh) [inline _data/errors.yml:53]: Should this be `vtr`? In the Biometric Pilot documentation for partners, we are telling them to use the `vtr` parameter, such as `vtr=["Pb","P1"]`

Comment (monfresh) [reply _data/errors.yml:53]: The oidc-sinatra sample app uses the `vtr` parameter in the auth request.

**Comment by tahineemay** [comment]
We can go ahead and close this PR, @jmhooper - confirmed as outdated and Team Melba will follow up

**Pull request closed without merging**